catalog-info.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: snyk-iac-rules
+  description: 
+  labels:
+    snyk.io/businessCriticality: Low
+    snyk.io/visibility: Public
```

Comment:
should be lower case

Reply:
Docs need updating then, I have just done so.
<img width="712" alt="image" src="https://user-images.githubusercontent.com/47144/148924625-0e47f090-a6e0-447d-91cf-456458b9a461.png">

Reply:
Thanks - I will do that!